app/models/school_group.rb

```diff
@@ -1,4 +1,11 @@
 class SchoolGroup < ApplicationRecord
   has_many :school_group_memberships
   has_many :schools, through: :school_group_memberships
+
+  has_many :vacancies
+
+  # TODO: This should ideally be a stored as column in the database table
```

Comment:
Not sure I agree with this as it's just copying source-of-truth for no real benefit.

Reply:
The thinking behind the TODO is that it would make it easier to lookup `SchoolGroups` by name if you ever wanted to do that